README.md

```diff
@@ -1,176 +1,166 @@
-# Debugger for .NET Core runtime
+# A Debugger for the .NET Core Runtime
 
-The debugger provides [GDB/MI](https://sourceware.org/gdb/onlinedocs/gdb/GDB_002fMI.html)
-and [VSCode Debug Adapter Protocol](https://microsoft.github.io/debug-adapter-protocol/)
-and allows to debug .NET apps under .NET Core runtime.  Also debugger allows debugging from
-command line (like as GDB).
+The NetCoreDbg debugger implements [GDB/MI](https://sourceware.org/gdb/onlinedocs/gdb/GDB_002fMI.html)
+and [VSCode Debug Adapter Protocol](https://microsoft.github.io/debug-adapter-protocol/) in a unified framework, allowing the debugging of .NET apps under the .NET Core runtime as well as facilitating debugging from the command line (such as in GDB).
 
 ## Copyright
 
-You can find licensing information in file [LICENSE](LICENSE), in root directory of Netcoredbg sources.
+You can find licensing information in the [LICENSE](LICENSE) file within the root directory of the NetCoreDbg sources.
 
 ## Usage
-More details about usage of NCDB you can find in [CLI](docs/cli.md) manual.
+Further details regarding the usage of NetCoreDbg can be found in the [CLI](docs/cli.md) manual.
 
+## Building from Source Code
 
-## Building from source code
-
-Currently Netcoredbg can be built on Linux, MacOS or Windows. Instructions for building Netcoredbg on each platform is shown below.
-
+Currently, NetCoreDbg can be built on Linux, MacOS, or Windows. Instructions for building NetCoreDbg on each platform is shown below.
 
 ### Unix
 
-Building of Netcoredbg requires Microsoft's .NET, so currently you can build Netcoredbg only in Linux. Microsoft supports at least few distributions, see details here: https://docs.microsoft.com/en-us/dotnet/core/install/linux
+NetCoreDbg's build requires Microsoft's .NET framework, and as such, NetCoreDbg can only be built in Linux. Microsoft supports a few distributions, the details of which can be found here: https://docs.microsoft.com/en-us/dotnet/core/install/linux.
```

Comment:
I think `.NET framework` here might be misleading as someone might think of .NET Framework. So it will be better as `Build of NetCoreDbg requires Microsoft's .NET, and as such, NetCoreDbg can only be built in Linux.`